subhub/tests/unit/test_subhub.py

```diff
@@ -16,6 +16,16 @@ class MockCustomer:
     object = "customer"
     subscriptions = [{"data": "somedata"}]
 
+    def props(self, cls):
```

Comment:
Is this copy paste of the previous code that I commented on?  If so why? If necessary, they all the same comments I said about that one

Reply:
This is in two instances yes, I have not seen your previous comment on this.  This is necessary to provide MockCustomer with a get() method.

Reply:
previous comment was why not `properties`?